common/src/main/java/com/wynntils/models/containers/containers/LootrunChestContainer.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright © Wynntils 2025.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.containers.containers;
+
+import com.wynntils.models.containers.Container;
+import com.wynntils.models.lootrun.type.LootrunLocation;
+import java.util.regex.Pattern;
+
+public class LootrunChestContainer extends Container {
+    private static final String TITLE_PATTERN = "^%s$";
+
+    private final LootrunLocation location;
+
+    public LootrunChestContainer(LootrunLocation location) {
+        super(Pattern.compile(String.format(TITLE_PATTERN, Pattern.quote(location.getContainerTitle()))));
+
+        this.location = location;
+    }
```

Comment:
```suggestion
    private static final String TITLE_PREFIX = "\uDAFF\uDFF2\uE00A\uDAFF\uDF6F";

    private final LootrunLocation location;

    public LootrunChestContainer(LootrunLocation location) {
        super(Pattern.compile(TITLE_PREFIX + location.getContainerTitle()));

        this.location = location;
    }
```

Then in `LootrunLocation` just give each type the ending character that differs